src/js/popup.js

```diff
@@ -51,11 +51,39 @@ function showSignUpPanel() {
 }
 
 
-function choosePanel(numRemaining, panelId, premium){
-  if (premium){
+function premiumFeaturesAvailable(premiumEnabledString) {
+  if (premiumEnabledString === "True") {
+    return Boolean(true);
+  }
+  else {
+    return Boolean(false);
+  }
+}
+
+function enablePremium(premium, premiumEnabledString) {
+  if (premium && premiumFeaturesAvailable(premiumEnabledString)) {
+    return Boolean(true);
```

Comment:
Not sure you need to wrap in `Boolean()`. You _could_ also reduce this to the following (although not convinced it's better/legible):

```js
return premium && premiumFeaturesAvailable(premiumEnabledString);
```